apps/meteor/server/startup/migrations/v306.ts

```diff
@@ -0,0 +1,22 @@
+import { Rooms } from '@rocket.chat/models';
+
+import { addMigration } from '../../lib/migrations';
+
+addMigration({
+	version: 305,
```

Comment:
Oops, this was supposed to be `version: 306` :grimacing: can you fix this (maybe in a new `chore` PR)?
Otherwise I believe this migration won't run
cc @dougfabris

Reply:
@matheusbsilva137 thanks for the heads-up. @ggazzo will handle it!